nbbuild/antsrc/org/netbeans/nbbuild/extlibs/MavenCoordinate.java

```diff
@@ -83,6 +83,11 @@ public String toMavenPath() {
     }
 
     public static boolean isMavenFile(String gradleFormat) {
+        if (gradleFormat.startsWith("http://") ||
+            gradleFormat.startsWith("https://") ||
+            gradleFormat.startsWith("file://")) {
+            return false;
+        }
         return gradleFormat.split(":").length > 2;
     }
```

Comment:
curious: what is this for? `hash space url space filename` did work before this change already, no?